package-lock.json

```diff
@@ -5328,7 +5328,7 @@
       "version": "3.1.1",
       "resolved": "https://registry.npmjs.org/anymatch/-/anymatch-3.1.1.tgz",
       "integrity": "sha512-mM8522psRCqzV+6LhomX5wgp25YVibjh8Wj23I5RPkPppSVSjyKD2A2mBJmWGa+KN7f2D6LNh9jkBCeyLktzjg==",
-      "dev": true,
+      "devOptional": true,
```

Comment:
what's the story with devOptional? I found this article: https://stackoverflow.com/questions/69871324/npm-install-creates-strange-differences-in-package-lock-json-between-different-m